packages/block-library/src/navigation/index.php

```diff
@@ -1617,10 +1617,16 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
 $rest_insert_wp_navigation_core_callback = 'block_core_navigation_' . 'update_ignore_hooked_blocks_meta'; // phpcs:ignore Generic.Strings.UnnecessaryStringConcat.Found
 
 /*
- * Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.5
- * that are not present in Gutenberg's WP 6.5 compatibility layer.
+ * Do not add the `block_core_navigation_update_ignore_hooked_blocks_meta` filter in the following cases:
+ * - If Core has added the `update_ignored_hooked_blocks_postmeta` filter already (WP >= 6);
```

Comment:
ugh I typo'd

```suggestion
 * - If Core has added the `update_ignored_hooked_blocks_postmeta` filter already (WP >= 6.6);
```

Reply:
It has now got to the point where I feel bad getting this merge stat once this gets approved 😂